deploy.sh

```diff
@@ -1,31 +1,4 @@
 #!/bin/bash
```

Comment:
no - i just wasn't sure about how it is hooked in into other systems, so i left it there for now